.changeset/config.json

```diff
@@ -0,0 +1,12 @@
+{
+  "$schema": "https://unpkg.com/@changesets/config@2.3.0/schema.json",
+  "changelog": ["@changesets/changelog-github", { "repo": "latticexyz/mud" }],
+  "commit": false,
+  "fixed": [],
+  "linked": [],
+  "access": "public",
+  "baseBranch": "main",
+  "updateInternalDependencies": "patch",
+  "ignore": []
```

Comment:
If there are any packages included in the pnpm-workspace.yml at the root we don't want to publish we should include them here.

If there are any packages NOT included in the pnpm-workspace.yml that we do want to publish I'm not sure immediately how other than publishing it seperately as a one off or adding it to workspace.yml